nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -97,7 +97,7 @@ let
         local size=$2
         local temp=$(mktemp)
         ${qemu}/bin/qemu-img create -f raw "$temp" "$size"
-        ${pkgs.e2fsprogs}/bin/mkfs.ext4 -L ${rootFilesystemLabel} "$temp"
+        ${cfg.host.pkgs.e2fsprogs}/bin/mkfs.ext4 -L ${rootFilesystemLabel} "$temp"
```

Comment:
If we use `cfg.host.pkgs` this often, perhaps a little `let hostPkgs = cfg.host.pkgs; in` would be cleaner.

Reply:
Done.